File: backend/app/schemas/session.py
from pydantic import BaseModel
# pydantic으로 세션 관련 스키마 작성 ( 자료형 제한 / 검증 과정)


# 새로운 세션이 생성되었을 때 클라이언트에게 반환되는 응답 모델
class SessionCreateResponse(BaseModel):
    session_id: str #생성된 세션 고유 식별자
    expires_in: int # 세션이 만료되기까지 남은 시간
    message: str # 세션 생성 상태를 설명하는 사용자 친화적인 메시지

# 이미 생성된 세션의 상태 정보를 조회할 때 반환되는 응답 모델
class SessionInfoResponse(BaseModel):
    session_id: str
    created_at: str # 처음 생성된 시간
    last_accessed_at: str # 세션이 마지막으로 사용된 시간

File: frontend/src/types/session.ts
export interface SessionCreateResponse {
  session_id: string;
  // 새로 생성된 세션의 고유 ID
  expires_in?: number;
  // 세션이 유지되는 시간 (초 단위, TTL)
  message?: string;
  // 세션 생성 관련 안내 메시지
}
// 새 세션을 만들때

export interface SessionInfoResponse {
  session_id: string;
  // 현재 세션 ID
  created_at?: string;
  // 세션 생성 시간
  last_accessed_at?: string;
  // 마지막으로 이 세션이 사용된 시간
}
// 기존 세션 조회할때
